codeforamerica/vita-min
# FYST-69 Removed skipped specs

## [FYST-69](https://codeforamerica.atlassian.net/browse/FYST-69)
## What was done?
 * Removed Skipped Tests
## How to test?
 * Specs should run and pass

[FYST-69]: https://codeforamerica.atlassian.net/browse/FYST-69?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
